icubam/backoffice/handlers/base.py

```diff
@@ -19,15 +19,14 @@ def initialize(self):
       self.root_path = '/{}/'.format(root)
     else:
       self.root_path = '/'
-    self.user = None
 
   def render(self, path, **kwargs):
     # This dictionary is updated by a PeriodicCallback in the
     # BackofficeApplication
     status = self.application.server_status
     super().render(
       path,
-      this_user=self.user,
+      this_user=self.current_user,
```

Comment:
perhaps I'm missing something, but self.current_user is never set.

Reply:
It is set by the `@tornado.web.authenticated` decorator.

Reply:
Ahhhh that's great, I'm double checking the documentation.
Perhaps one little comment about it in the get_current_user method?

Reply:
See https://www.tornadoweb.org/en/stable/guide/security.html#user-authentication

Reply:
Sure, I'll add a comment.

Reply:
I saw it later. Apparently we don't even need to inject the self.current_user into the template as this_user since the user is already injected according to the docs.

But this can be fixed in a follow-up CL (i.e., cleaning up the templates in favour of current_user instead of this_user). Can you add a TODO for that?

Thanks a lot for the nice fix.

Reply:
Ah ah, I also realized while sending you the pointer :)

I can do it in this PR, it is morally part of the same change.

Reply:
cool ! Good job ! Feel free to do it either in this or in a subsequent PR.